tests/run/test_grammar.py

```diff
@@ -486,6 +486,7 @@ class Cbad2(C):
                 x: int
                 x.y: list = []
 
+    @skip("Not currently supported: #3839")
```

Comment:
Better be explicit about what we're referring to.
```suggestion
    @skip("Not currently supported: https://github.com/cython/cython/issues/3839")
```